jerry-core/api/jerry-debugger.c

```diff
@@ -116,3 +116,82 @@ jerry_debugger_cleanup (void)
   }
 #endif /* JERRY_DEBUGGER */
 } /* jerry_debugger_cleanup */
+
+/**
+ * Sets whether the engine should wait and run a source.
+ *
+ * @return FAIL - if the source is not received or parsed
+ *         SUCCESS - if the client source has been received and processed
+ */
+jerry_debugger_wait_and_run_type_t
+jerry_debugger_wait_and_run_client_source (jerry_value_t *return_value) /**< [out] parse and run return value */
+{
+#ifdef JERRY_DEBUGGER
```

Comment:
Please move `*return_value = jerry_create_undefined ();` before the ifdef so the `return_value` is always filled with something valid.